Correct external DB instructions for foreman-{deb,el} (#2396)
theforeman/foreman-documentation: guides/common/modules/con_migrating-from-internal-databases-to-external-databases.adoc

To migrate from the default internal databases to external databases, you must complete the following procedures:

ifndef::foreman-deb[]
. xref:preparing-a-host-for-external-databases_{context}[].

> // TODO: a {RHEL} vs an {EL}
> Prepare a {RHEL} 8 server to host the external databases.

Reviewer: ```suggestion
Prepare your {RHEL} 8 server to host the external databases.
```
